htdocs/theme/eldy/global.inc.php

```diff
@@ -358,7 +358,7 @@
 	padding-right: 5px;
 	background-color: transparent;
 	color: var(--colortext) !important;
-	border: 2px solid var( --butactionbg) !important;
+	border: 2px solid rgb(<?php echo $butactionbg ?>) !important;
```

Comment:
You must keep the var( --butactionbg)   (everywhere)
and add only at one place the <?php echo $butactionbg ?> where the var  butactionbg is defined.

Reply:
take a look at this https://www.youtube.com/watch?v=IHaT_rjC2gM this guy explain a good usage of css var

Reply:
@atm-john @eldy Sory ok for me i change

it's done